### ISSUE TITLE
Feat req: export specific register value

### ISSUE THREAD

**Issue by flespark** [mcu-debug/svd-viewer#29]
Type: Feature Request
Hi，Dear maintainers
In my embedded develop scene，I found that after changed some MCU config, one peripheral of MCU often can not work as previous any more, even though I repeatedly checked the modified code have not touch any relative register of the peripheral. So, to confirm if any relative register change, I have to debug old code, copy out all relative register one by one; then compile and debug new code,  halt at same position,  dump the registers again,  copy two version registers to a diff tool for compare. Maybe a function export all or specific peripheral register will be a very facility for such scene debug

**Comment by cblack-f3** [comment]
Some sort of "Dump peripheral registers" would be great!

**Comment by 0Grit** [comment]
Yes, this would make it so much easier to dump and diff between two devices or firmware builds.
It's an essential debug technique when dealing with all sorts of power, clock, and peripheral problems.
